homeassistant/components/proxmox/switch.py

```diff
@@ -0,0 +1,77 @@
+"""Support for switches to turn on/off Proxmox VE VMs/Containers."""
+import homeassistant.components.proxmox as proxmox
+from homeassistant.components.switch import SwitchDevice
+
+
+async def async_setup_platform(
+        hass, config, async_add_entities, discovery_info=None):
+    """Set up Proxmox VE switch."""
+    nodes = hass.data[proxmox.DATA_PROXMOX_NODES]
+    control = hass.data[proxmox.DATA_PROXMOX_CONTROL]
+    sensors = []
+
+    for node_name in nodes.keys():
+        item = nodes[node_name]
```

Comment:
Maybe there should be an option in the config to state explicitly that you want to configure the switches. Because turning off the VMs is quite a powerful action.